### ISSUE TITLE
Wallet never syncs

### ISSUE THREAD

**Issue by blastoplast** [hempcoin-project/THC#14]
I downloaded this wallet over 2 weeks ago and it has been syncing constantly since that time. It has been stuck on "55030 blocks remaining" for several days now. The log shows several ERROR CheckProofOfStake instances. I have yet to actually open the wallet successfully and see my coins in it.

**Comment by blastoplast** [comment]
![wallet](https://user-images.githubusercontent.com/33377110/34387474-98429296-eaeb-11e7-8f03-c475ebcbb7cd.JPG)

**Comment by blastoplast** [comment]
Still hasn't moved.

**Comment by withthelemons** [comment]
Please delete everything in %appdata%/Hempcoin EXCEPT FOR WALLET.DAT and try again.

**Comment by withthelemons** [comment]
Also make sure your computer time is correct

**Comment by blastoplast** [comment]
Ok, I deleted everything except wallet.dat and fired it back up. Now it stops during the loading process and gives me an error that says "Error loading blkindex.dat."  If it helps, I still have the old files. I just moved them to a new folder on the desktop in case I ended up needing them again.

**Comment by finleyexp** [comment]
> "Error loading blkindex.dat."
This is the error I'm also receiving.

**Comment by 0xfff** [comment]
make sure you have  v2.0.1.0
also,make sure you have a `Hempcoin.conf` file in your `AppData/Roaming/Hempcoin` directory (this is on windows)
I have this in my .conf:
```
addnode=54.186.62.109
addnode=88.208.236.108
addnode=198.50.132.65
addnode=69.10.58.157
addnode=115.236.88.86:21054
addnode=24.224.89.67:21054
addnode=54.186.62.109
addnode=69.10.58.157
addnode=192.168.0.13
addnode=198.27.106.7
addnode=162.243.1.45
addnode=162.209.109.81
addnode=83.183.247.151
addnode=24.69.200.51
addnode=82.21.238.186
addnode=54.186.62.109
addnode=88.208.236.108
addnode=198.50.132.65
addnode=69.10.58.157
addnode=108.49.185.3:57981
addnode=31.31.78.40:21054
addnode=46.167.245.107:21054
addnode=125.121.109.74:7020
addnode=75.130.162.248:55925
addnode=85.236.188.28:52026
addnode=188.162.200.197:15667
addnode=129.21.131.103:21054
addnode=195.178.92.11:53252
addnode=195.181.208.162:21054
addnode=125.121.109.74:7339
addnode=129.21.131.183:48072
addnode=40.112.185.197:1336
addnode=[2001:15e8:110:2fa2::21]:21054
addnode=[2a02:2b88:2:1::53f7:1]:21054
addnode=[2001:0:5ef5:79fb:1c58:38dd:7eea:7c98]:52534
addnode=[2001:0:5ef5:79fb:1c58:38dd:7eea:7c98]:58960
addnode=[2001:0:5ef5:79fb:8c6:31b2:a695:95e1]:59569
addnode=[2001:0:9d38:6ab8:34a6:c629:8c13:a7a9]:51279
addnode=[2001:0:9d38:6abd:10b6:faa8:aa13:43e3]:65089
addnode=[2001:0:9d38:6abd:1857:28d9:aa13:43e3]:49618
addnode=[2001:0:9d38:78cf:1813:905e:9c92:8961]:6225
addnode=[2001:0:9d38:78cf:87d:8f1e:9c92:8961]:21054
addnode=[2001:0:9d38:90d7:1c05:1e65:ab28:f82d]:21054
addnode=[2001:0:9d38:953c:18cd:29a1:e71f:a6bc]:51096
addnode=[2001:0:9d38:953c:2428:16c5:b47d:5d07]:49366
addnode=[2001:0:9d38:953c:182a:6d84:8286:92b5]:21054
addnode=[2001:0:9d38:953c:2025:23ce:b356:c6a3]:42490
addnode=[2001:0:9d38:953c:30d0:f7db:8c13:a7a9]:55789
addnode=[2001:0:9d38:953c:68:30be:4470:8932]:53690
addnode=[2001:0:9d38:953c:856:922:b341:314a]:52413
addnode=[2601:8c5:0:7b39:49ba:5f6:27c2:5470]:52988
addnode=[2607:fea8:e380:e8c:7478:5d52:622e:ada4]:64786
```

**Comment by 0xfff** [comment]
i'm synced now at block 2006116
```
getinfo
￼
{
"version" : "v2.0.1.0",
"protocolversion" : 80020,
"walletversion" : 60000,
"balance" : ,
"newmint" : 0.00000000,
"stake" : 0.00000000,
"blocks" : 2006116,
"timeoffset" : -18,
"moneysupply" : 230393805.68869001,
"connections" : 9,
"proxy" : "",
"ip" : "",
"difficulty" : {
"proof-of-work" : 2096.93774662,
"proof-of-stake" : 16710.31456574
},
"testnet" : false,
"keypoololdest" : 1513886677,
"keypoolsize" : 100,
"paytxfee" : 0.00100000,
"mininput" : 0.02000000,
"unlocked_until" : 0,
"errors" : ""
}
```

**Comment by withthelemons** [comment]
Should be fixed in v2.1.0 #16
